Test failures in build step after last commits

Hey Egor!

I tried to build the new application after your last changes (like replace deprecated API, metadata topics now are closed correctly and dependencies were updated), but maven returns the message "There are test failures". It seems that something is wrong with the SimpleCounterTest class (NullPointerExcepion). I will write below a peace of the error and if you need something additional just ask. 

Tks.

====> TEST-ru.cinimex.exporter.prometheus.metrics.SimpleCounterTest.xml:

"<testcase name="createValidSimpleCounter" classname="ru.cinimex.exporter.prometheus.metrics.SimpleCounterTest" time="0">
    <error type="java.lang.NullPointerException">java.lang.NullPointerException
	at ru.cinimex.exporter.prometheus.metrics.SimpleCounterTest.createValidSimpleCounter(SimpleCounterTest.java:28)"

====> Maven:

" [ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 0 s <<< FAILURE! - in ru.cinimex.exporter.prometheus.metrics.SimpleCounterTest
[ERROR] createValidSimpleCounter  Time elapsed: 0 s  <<< ERROR!
java.lang.NullPointerException
        at ru.cinimex.exporter.prometheus.metrics.SimpleCounterTest.createValidSimpleCounter(SimpleCounterTest.java:28)

[INFO] Running ru.cinimex.exporter.prometheus.metrics.SimpleGaugeTest
[INFO] Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 0.016 s - in ru.cinimex.exporter.prometheus.metrics.SimpleGaugeTest
[INFO]
[INFO] Results:
[INFO]
[ERROR] Errors:
[ERROR]   SimpleCounterTest.createValidSimpleCounter:28 » NullPointer
[INFO]
[ERROR] Tests run: 51, Failures: 0, Errors: 1, Skipped: 0
[INFO]
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  13.472 s
[INFO] Finished at: 2021-02-28T17:00:48-03:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.22.1:test (default-test) on project mq-java-exporter: There are test failures.
[ERROR]
[ERROR] Please refer to C:\Users\Alex Alves\Desktop\Prometheus2\exporterJava\target\surefire-reports for the individual test results.
[ERROR] Please refer to dump files (if any exist) [date].dump, [date]-jvmRun[N].dump and [date].dumpstream.
[ERROR] -> [Help 1]
[ERROR]
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR]
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException"

--- a/pom.xml
+++ b/pom.xml
@@ -11,7 +11,7 @@
         <jdk.version>1.8</jdk.version>
         <mq.allclient.version>9.2.1.0</mq.allclient.version>
         <snakeyaml.version>1.28</snakeyaml.version>
-        <prometheus.version>0.10.0</prometheus.version>
+        <prometheus.version>0.9.0</prometheus.version>
         <log4j.version>2.14.0</log4j.version>
         <junit.jupiter.version>5.7.1</junit.jupiter.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>

--- a/src/test/java/ru/cinimex/exporter/testutils/Utils.java
+++ b/src/test/java/ru/cinimex/exporter/testutils/Utils.java
@@ -3,7 +3,7 @@ package ru.cinimex.exporter.testutils;
 import io.prometheus.client.CollectorRegistry;
 
 public class Utils {
-    public static double getPrometheusMetricValue(CollectorRegistry registry, String metricName,
+    public static Double getPrometheusMetricValue(CollectorRegistry registry, String metricName,
                                                   String[] labelNames, String[] labelValues) {
         return registry.getSampleValue(metricName, labelNames, labelValues);
     }

